itests/wdpost_test.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 func TestWindowedPost(t *testing.T) {
-	kit.Expensive(t)
+	//kit.Expensive(t)
```

Comment:
What's this about? Should we delete this line?

Reply:
Oops, that was me debugging test failure thanks for busting me.